src/docs/v10-vs-v11-retrospective.md

```diff
@@ -0,0 +1,643 @@
+development of version 11 was launched during a meeting at the Lawrence Berkeley National Laboratory in December 2023,
+and the development took place in a fork of the nmdc-schema called berkeley-schema-fy24. This is the first time a major
+update had comprehensive planning and development in a fork. A subsequent update in the same style has not planned yet
+but is not ruled out.
+
+Motivations for the Berkeley schema development included:
+
+- First class representations of "omics" methods beyond nucleic acid analysis, like mass-spectrometry based
+  proteomics and metabolomics. Within the Berkeley schema development team, this effort has gone by names like
+  "replacing `OmicsProcessing` with `DataGeneration`".
+- Renaming and refactoring of the processes that model the processing of data, i.e. bioinformatics workflows.
+- A continuation of bipartite (material->process->material->process) modeling for the processing of samples, following
+  the bipartite modeling used in the bioinformatics workflows (data->process->data).
+- Continuation of using example data files as unit tests of the schema
+- Switching numerous string-range slots to enumeration-range slots.
+- A requirement that all data records assert their own type and the addition of support for polymorphic aggregations of
+  data
+- Establishing schema development and modeling best practices and sharing domain understanding between subteams
+
+See also: https://github.com/microbiomedata/nmdc-schema/releases/tag/v11.0.0
+
+## Conventions:
+
+- `this back-ticked font` is used to indicate the verbatim names of elements from the schema and variables, functions,
+  etc. in code
+- Things that appear in the back-ticked font and use `UpperCamelCase` are schema classes unless otherwise noted.
+- Things that appear in the back-ticked font and use `lower_snake_case` are schema slots unless otherwise noted.
+- Other LinkML schema [naming conventions](https://linkml.io/linkml/schemas/linter.html#standard-naming) are not
+  discussed here.
+- A dotted notation in the back-ticked font, with elements in UpperCamelCase, like `Vehichle.MotorVehicle.PassengerCar`
+  indicates a class hierarchy. `PassengerCar is_a MotorVehicle` and `MotorVehicle is_a Vehicle`.
+
+## Refactoring of Classes
+
+This major refactoring increases the number of **classes** in the schema from 70 to 75, but decreases the number of root
+classes from 19 to 17 due to better organization.
+
+_You won't find any classes designated as root classes in the schema,
+but they can be inferred by the absence of any `is_a` assertions on the class. Root classes do not inherit anything
+from any superclass. A routine part of schema maintenance is looking for opportunities to group and generalize similar
+root classes._
+
+### Table of classes removed, added or refactored in v10 to v11 migration
+
+| Removed legacy class                  | New/Replacement class from Berkeley schema | Selected legacy parents     |
```

Comment:
I don't know what the "selected legacy parents" are trying to convey here?

Reply:
I also am not sure why "selected legacy parents" is the chosen words.. can it just be "legacy parent"?

Reply:
It's "selected" because I didn't fill in all parents for all rows

Reply:
ooooo ok. IDK how else to convey that so people like me aren't confused.. so probably fine

Reply:
I guess it deserves a brief explanation in the text

Reply:
https://github.com/microbiomedata/nmdc-schema/pull/2227/commits/08d5cada228a4967342a85070125cb0ff7971961